ui/src/app/subnets/views/SubnetDetails/SubnetDetailsHeader/SubnetActionForms/components/EditBootArchitectures/EditBootArchitectures.tsx

```diff
@@ -34,7 +52,39 @@ export const MapSubnet = ({
   }
 
   const closeForm = () => setActiveForm(null);
-  return <Button onClick={closeForm}>Close</Button>;
+  return (
+    <FormikForm<FormValues>
+      buttonsBordered={false}
+      cleanup={cleanup}
+      errors={errors}
+      initialValues={{
+        disabled_boot_architectures: subnet.disabled_boot_architectures,
+      }}
+      onCancel={closeForm}
+      onSaveAnalytics={{
+        action: "Edit boot architectures",
+        category: "Subnet details",
+        label: "Edit boot architectures",
+      }}
+      onSubmit={(values) => {
+        dispatch(cleanup());
+        dispatch(
+          subnetActions.update({
+            disabled_boot_architectures:
+              values.disabled_boot_architectures.join(", "),
+            id: subnet.id,
+          })
+        );
+      }}
+      onSuccess={closeForm}
+      saved={saved}
+      saving={saving}
+      submitLabel="Edit boot architectures"
```

Comment:
I think this would make more sense to say "Save" (that's what the legacy version has too).